test/views/MentionsAndSentiments/test.index.js

```diff
@@ -120,11 +120,19 @@ describe('<MentionsAndSentiments />', () => {
     'text': 'Sample Company'
   };
 
-  it('Orders results by number of "mentions"', () => {
+  it('Orders results by number of mentions', () => {
     let wrapper = shallow(<MentionsAndSentiments query={query_sample} mentions={mentions_sample} />);
     let firstMention = wrapper.find(Accordion).nodes[0].props.header.props.children[0].props.children;
     let secondMention = wrapper.find(Accordion).nodes[1].props.header.props.children[0].props.children;
     assert.equal(firstMention, 'Sample Company + Company Two');
     assert.equal(secondMention, 'Sample Company + Company One');
   });
+
+  it('Reports number of mentions, not number of days a mention occured', () => {
```

Comment:
no reason to mention the previous behavior - that concept should disappear. instead use a test name that explains the expected behavior - maybe something like "calculates a summation of positive, negative, and neutral mentions"